www/js/main.js

```diff
@@ -3129,7 +3129,7 @@ function overlayMap( callback ) {
 function debugWU() {
 	var popup = "<div data-role='popup' id='debugWU' class='ui-content ui-page-theme-a'><table class='debugWU'>";
 
-	if ( typeof controller.settings.wtdata === "object" ) {
+	if ( typeof controller.settings.wtdata === "object" && Object.keys( controller.settings.wtdata ).length > 0 ) {
```

Comment:
I think it would be better to reverse the information in this popup, i.e., put the most important info first
1) start with Last Reboot & Reason, then,
2) Last Weather Call & Last Successful Call, then,
3) "Success" or  Weather Error message, and finish with,
4)  wtdata as a prettified json dump into text field.

This is because if using ET then the key parameter is solarRadiation so a dump of wtdata is probably more useful. I think displaying the Weather Error could resolve a lot of the user's "diagnostic" concerns.